src/test/java/store/controller/CategoryControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package store.controller;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CategoryControllerTest {
+
+    @InjectMocks
+    private CategoryController categoryController;
+
+    @Mock
+    private CategoryController controller;
+
```

Comment:
I think it's not  correct to create @Mock and @InjectMocks. Only @InjectMocks is needed